dev-python/jaraco-logging/jaraco-logging-1.5.2.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} pypy{,3} )
+
+inherit distutils-r1
+
+MY_PN="${PN/-/.}"
+DESCRIPTION="Additional facilities to supplement Python's stdlib logging module"
+HOMEPAGE="https://github.com/jaraco/jaraco.logging"
+SRC_URI="mirror://pypi/${PN:0:1}/${MY_PN}/${MY_PN}-${PV}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
+
+RDEPEND="
+	dev-python/namespace-jaraco[${PYTHON_USEDEP}]
```

Comment:
Where in https://github.com/jaraco/jaraco.logging/blob/1.5.2/setup.py is this pulled in and how?

Reply:
Please Read The Friendly Manual: https://wiki.gentoo.org/wiki/Project:Python/Namespace_packages